### PR TITLE
Feature liq 116 post params in body

### PR DESCRIPTION
**Pull request by filip-prime**

### CODE REVIEW

File: src/Lykke.Service.BitfinexAdapter.Services/ExecutionHarvester/BitfinexExecutionHarvester.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+using Lykke.Service.BitfinexAdapter.Core.Domain.Trading;
+using Lykke.Service.BitfinexAdapter.Core.Domain.WebSocketClient;
+using Lykke.Service.BitfinexAdapter.Core.Handlers;
+using Lykke.Service.BitfinexAdapter.Core.Utils;
+using Lykke.Service.BitfinexAdapter.Core.WebSocketClient;
+using System.Threading.Tasks;
+
+namespace Lykke.Service.BitfinexAdapter.Services.ExecutionHarvester
+{
+    public sealed class BitfinexExecutionHarvester : IStartable, IStopable
```
Comment (pavel-khritonenko) [inline src/Lykke.Service.BitfinexAdapter.Services/ExecutionHarvester/BitfinexExecutionHarvester.cs]: Don't use IStartable feature (it's AutoFac's issue, use StartupManager instead)

File: src/Lykke.Service.BitfinexAdapter.Services/OrderBooksHarvester/OrderBooksHarvesterBase.cs [hunk 2]
```diff
@@ -97,12 +95,12 @@ protected void RechargeHeartbeat()
             while (!CancellationToken.IsCancellationRequested)
             {
                 var msgInSec = _lastSecPublicationsNum / period;
-                var pubInSec = _publishedToRabbit / period;
+                var pubInSec = _totalOrderbooksPublishedToRabbit / period;
                 await Log.WriteInfoAsync(nameof(OrderBooksHarvesterBase),
-                    $"Receive rate from {ExchangeName} {msgInSec} per second, publish rate to " +
-                    $"RabbitMq {pubInSec} per second", string.Empty);
+                    $"Receive rate from {Constants.BitfinexExchangeName} {msgInSec} per second, publish rate to " +
+                    $"RabbitMq {pubInSec} per second.", string.Empty);
```
Comment (pavel-khritonenko) [inline src/Lykke.Service.BitfinexAdapter.Services/OrderBooksHarvester/OrderBooksHarvesterBase.cs]: Is it correct overload of WriteInfo? Seems like you log empty string.

File: src/Lykke.Service.BitfinexAdapter.Services/ShutdownManager.cs [hunk 3]
```diff
@@ -16,12 +19,16 @@ public class ShutdownManager : IShutdownManager
     {
         private readonly ILog _log;
         private readonly List<IStopable> _items = new List<IStopable>();
-        private readonly ExchangeBase _exchange;
+        private readonly BitfinexOrderBooksHarvester _orderBooksHarvester;
+        private BitfinexAdapterSettings _settings;
+        private IComponentContext _container;
 
-        public ShutdownManager(ILog log, ExchangeBase exchange)
+        public ShutdownManager(ILog log, BitfinexOrderBooksHarvester orderBooksHarvester, BitfinexAdapterSettings settings, IComponentContext container)
```
Comment (pavel-khritonenko) [inline src/Lykke.Service.BitfinexAdapter.Services/ShutdownManager.cs]: The second parameter could be just `IEnumerable<IStoppable>`

**Review by pavel-khritonenko — APPROVED** [review]

File: src/Lykke.Service.BitfinexAdapter.Services/ExecutionHarvester/BitfinexExecutionHarvester.cs (hunk 1 above)
Comment (filip-prime) [reply src/Lykke.Service.BitfinexAdapter.Services/ExecutionHarvester/BitfinexExecutionHarvester.cs]: Done. BitfinexExecutionHarvester instantiation and starting is now handled by StartupManager

File: src/Lykke.Service.BitfinexAdapter.Services/OrderBooksHarvester/OrderBooksHarvesterBase.cs (hunk 2 above)
Comment (filip-prime) [reply src/Lykke.Service.BitfinexAdapter.Services/OrderBooksHarvester/OrderBooksHarvesterBase.cs]: Done. Log message used placed at the wrong place - its a valid overload but context and info params were swapped.

File: src/Lykke.Service.BitfinexAdapter.Services/ShutdownManager.cs (hunk 3 above)
Comment (filip-prime) [reply src/Lykke.Service.BitfinexAdapter.Services/ShutdownManager.cs]: Done. Refactored.

**Pull request merged**